x-pack/plugins/observability_solution/observability/public/pages/alert_details/alert_details.tsx

```diff
@@ -175,13 +178,20 @@ export function AlertDetails() {
         <AlertDetailContextualInsights alert={alertDetail} />
         <EuiSpacer size="l" />
         {rule && alertDetail.formatted && (
-          <AlertDetailsAppSection
-            alert={alertDetail.formatted}
-            rule={rule}
-            timeZone={timeZone}
-            setAlertSummaryFields={setSummaryFields}
-            ruleLink={http.basePath.prepend(paths.observability.ruleDetails(rule.id))}
-          />
+          <>
+            <AlertDetailsAppSection
+              alert={alertDetail.formatted}
+              rule={rule}
+              timeZone={timeZone}
+              setAlertSummaryFields={setSummaryFields}
+              ruleLink={http.basePath.prepend(paths.observability.ruleDetails(rule.id))}
+            />
+            <EuiSpacer size="l" />
+            <AlertHistoryChart
+              alert={alertDetail.formatted}
+              rule={rule as unknown as CustomThresholdRule}
```

Comment:
To avoid this casting, I tried to parameterize the function that fetches the rule ([useFetchRule](https://github.com/elastic/kibana/blob/main/x-pack/plugins/observability_solution/observability/public/hooks/use_fetch_rule.ts#L33)) but I am not sure why it does not work, if you have any suggestions, please let me know:

![image](https://github.com/elastic/kibana/assets/12370520/acfe00a7-9667-4b32-bfc7-9d1bc0720b83)